src/OrbitGl/App.h

```diff
@@ -347,11 +347,15 @@ class OrbitApp final : public DataViewFactory,
   orbit_base::Future<void> RetrieveModulesAndLoadSymbols(
       absl::Span<const orbit_client_data::ModuleData* const> modules) override;
 
+  enum class SymbolLoadingResult {
+    kSymbolsLoadedSuccessfully,
+    kCancelled,
+  };
```

Comment:
Just very personal opinion. If I only look at the code here, I feel a little bit strange that there is no error case in the symbol loading result. Maybe add a comment here.

Reply:
That is a good point. I am going to change this to `SymbolErrorHandlingResult` because this is more accurate and then it also makes more sense, right?

Reply:
Actually now called: `SymbolLoadingAndErrorHandlingResult`